File: 프로그래머스/0/120885. 이진수 더하기/이진수 더하기.js
function binaryCalculate(bin1, bin2) {
    const int1 = parseInt(bin1, 2);
    const int2 = parseInt(bin2, 2);
    const result = int1 + int2;
    
    return result.toString(2);
}

function solution(bin1, bin2) {
    let answer = '';

    answer = binaryCalculate(bin1, bin2);
    
    return answer;
}

File: 프로그래머스/0/120885. 이진수 더하기/이진수 더하기.java
class Solution {
    public String solution(String bin1, String bin2) {
        String answer = String.valueOf(Integer.toBinaryString((Integer.valueOf(bin1,2)+Integer.valueOf(bin2,2))));
        int sum = 8;
        sum /= 1+ 1;
        System.out.println(sum);
        
        return answer;
    }
}